src/main.js

```diff
@@ -328,20 +388,26 @@ async function main() {
         }
         if (createUpdatePipelineEndpoint == '' &&
             createUpdatePipelineRegion == '' && createUpdatePipelineNeptuneDatabaseName == '') {
-            console.error('AWS pipeline: is required a Neptune endpoint, or a Neptune database name and region.');
+            msg = 'AWS pipeline: is required a Neptune endpoint, or a Neptune database name and region.';
+            console.error(msg);
+            loggerLog(msg);            
             process.exit(1);
         }
         if (createUpdatePipelineEndpoint == '' &&
             !createUpdatePipelineRegion == '' && createUpdatePipelineNeptuneDatabaseName == '') {
-            console.error('AWS pipeline: a Neptune database name is required.');
+            msg = 'AWS pipeline: a Neptune database name is required.';
+            console.error(msg);
+            loggerLog(msg);            
             process.exit(1);
         }
         if (createUpdatePipelineEndpoint == '' &&
             createUpdatePipelineRegion == '' && !createUpdatePipelineNeptuneDatabaseName == '') {
-            console.error('AWS pipeline: a Neptune database region is required.');
+            msg = 'AWS pipeline: a Neptune database region is required.';
+            console.error(msg);
+            loggerLog(msg);            
             process.exit(1);
         }        
-        if (createUpdatePipelineEndpoint != '') {
+        if (createUpdatePipelineEndpoint != '' && createUpdatePipelineRegion == '') {
```

Comment:
I think this actually introduced a bug where we are now required to input `--createUpdatePipelineRegion` to get the utility to work. The value for createUpdatePipelineRegion is default `us-east-1` so this code will never execute (unless the user specifies a blank value). We should ask Andrea if he wants us to remove the default value.

Reply:
Currently, `--createUpdatePipelineRegion` is needed when creating a Database GraphQL API but when creating an Analytics one it doesn't. Trying to create an API with an Analytics endpoint without `--createUpdatePipelineRegion` runs successfully in the terminal, but the API is not created when you look at the AppSync page. It seems like Analytics is missing a statement that checks if `createUpdatePipelineRegion` is empty.

Reply:
For analytics db without specifying --createUpdatePipelineRegion it should be using the default region us-east-1 and I bet it's creating the API in that region but you're looking at the us-west-2 region.

I think we should remove the default region and then we need to fix the logic that is parsing the region out of the db URL to handle analytics URL which is formatted differently from neptune db url:

achild-db-neptune-1.cluster-cjiepzx2kerx.us-west-2.neptune.amazonaws.com:8182
VS
g-hgb6zpdbh5.us-west-2.neptune-graph.amazonaws.com:8182

Reply:
@sophiadt I did a [commit](https://github.com/aws/amazon-neptune-for-graphql/commit/58c248a0b8d78cd8c245bc0515f747f756d6d372) which should hopefully fix the issue. Can you try it out? This fix should mean that you don't need to specify the --create-update-aws-pipeline-region or --create-update-aws-pipeline-neptune-database-name options for analytics or neptune db anymore.

Reply:
Okay I tested it and it worked thanks!